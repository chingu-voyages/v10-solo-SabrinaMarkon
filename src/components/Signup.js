import React, { Component } from 'react';

export default class Signup extends Component {
  constructor(props) {
    super(props);


  }

  render() {
    return (
      <div>
        <h1>Signup</h1>
      </div>
    )
  }
}