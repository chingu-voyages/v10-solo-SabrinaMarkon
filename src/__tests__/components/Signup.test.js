import React from 'react';
import { shallow } from 'enzyme';

// components
import Signup from '../components/Signup';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Signup />);
});

describe('Testing Signup component', () => {
  it('should render an h1 heading with the correct text', () => {
    expect(wrapper.find('h1').text().toEqual('Signup'));
  });
});