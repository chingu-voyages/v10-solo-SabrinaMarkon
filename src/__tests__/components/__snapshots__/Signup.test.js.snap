// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Signup component it should match the Signup snapshot 1`] = `undefined`;
